user_management/ui/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from django.http import Http404
+from django.utils.translation import ugettext_lazy as _
+
+
+class InvalidExpiredToken(Http404):
+    """Exception to confirm an account."""
+    message = _('Invalid or expired token.')
```

Comment:
I really don't think this should be a `404`. It looks more like a `401` or maybe a `403` to me.